flow-typed/box-ui-elements.js

```diff
@@ -733,6 +733,8 @@ type AppActivityItems = {
 type FeedItem = Comment | Task | TaskNew | BoxItemVersion | AppActivityItem;
 type FeedItems = Array<FeedItem>;
 
+type FeedItemType = 'task' | 'comment';
```

Comment:
Could we make use of the existing constants? `COMMENT_TYPE_DEFAULT` and `COMMENT_TYPE_TASK`?

Reply:
Those constants are obsolete and will be removed in a future clean up PR